custom_model_runner/datarobot_drum/drum/adapters/model_adapters/python_model_adapter.py

```diff
@@ -763,6 +766,50 @@ def chat(self, completion_create_params, model, association_id):
         else:
             return self._custom_hooks.get(CustomHooks.CHAT)(completion_create_params, model)
 
+    def get_supported_llm_models(self, model):
+        """
+        Return list of LLM models supported by this custom model.
+        Usually, the trigger is a call to OpenAI models.list() (/v1/models, or /models)
+
+        If custom.py defines get_supported_llm_models(), use its returned list.
+        If that hook is not defined, return the model defined in LLM_ID runtime parameter.
+        If neither is present, return an empty list.
+
+        Parameters
+        ----------
+        model
+
+        Returns
+        -------
+
```

Comment:
No return? 
Actually it returns some structure, maybe even worth to prepare a structure type (datatype?) to make it self descriptive